methcomp/blandaltman.py

```diff
@@ -181,74 +182,77 @@ def plot(
         pkws.update(point_kws or {})
 
         # Get parameters
-        mean, mean_ci = self.result["mean"], self.result["mean_CI"]
+        mean, mean_CI = self.result["mean"], self.result["mean_CI"]
+        loa_upper, loa_upper_CI = self.result["loa_upper"], self.result["loa_upper_CI"]
+        loa_lower, loa_lower_CI = self.result["loa_lower"], self.result["loa_lower_CI"]
+        sd_diff = self.result["sd_diff"]
 
         # individual points
         ax.scatter(self.mean, self.diff, **pkws)
 
         # mean difference and SD lines
         ax.axhline(mean, color=color_mean, linestyle="-")
-        ax.axhline(mean + self.loa_sd, color=color_loa, linestyle="--")
-        ax.axhline(mean - self.loa_sd, color=color_loa, linestyle="--")
+        ax.axhline(loa_upper, color=color_loa, linestyle="--")
```

Comment:
should let this alpha be `self.loa_linestyle`, initialized by default `"--"` in constructor

Reply:
added this now as a plot() argument since it's only graphically relevant